### DISCUSSION TITLE
[国际服]萨米肉鸽N15刷精二开局bug

### DISCUSSION THREAD

**Discussion by neon1300** [MaaAssistantArknights/MaaAssistantArknights#11096] [Q&A]
似乎这里有bug, 尽管难度调成了MAX 或者 15 (两者都试了), 但是在开始游戏时 MAA会先把难度调回N0 (也就是Braving Nature难度)
版本: 公测版
UI Version: v5.9.0
Core Version: v5.9.0
Build Time: 2024/11/4 03:22:21

自动肉鸽设定:
![Screenshot 2024-11-07 173532](https://github.com/user-attachments/assets/345cd517-2bd5-4d54-8582-9cab2179027f)
![Screenshot 2024-11-07 173603](https://github.com/user-attachments/assets/ebdf64f9-54df-4d90-b338-04f04d0fe1da)

感谢🙇‍

**Comment by Daydreamer114** [comment]
请在主仓库中提交 issue

**Comment by ABA2396** [comment]
刷开局就是n0刷，然后去n15看有没有刷完。

**Reply by neon1300** [reply]
没看懂, 刷精二开局要怎么中途从n0换成n15? 没法换啊。

**Reply by ABA2396** [reply]
刷的难道不是职业队的开局直升吗

**Reply by neon1300** [reply]
是啊, 我要n15精二EW开局, 它给我搞成了n0精二EW开局...

**Comment by Alan-Charred** [comment]
您好，刷开局模式的逻辑是建立在刷开局奖励的基础上的。无论您怎么设置，都会先打一把 N0，达到第三层时再重开，之后切换回您所设置的难度，尝试凹开局。

目前 MAA 并不支持纯凹开局精二。考虑到萨卡兹肉鸽开始，职业分队开局必有精二的情况，目前也没有优化这个功能的计划。

您可以发个 Issue 来提出想要改进的功能，如果有更多用户反应希望有这方面的功能改进的话，在未来重构相关功能时我们会进行尝试。就我本人看来，先来一把 N0 来换个更好的开局，未尝不可～

**Reply by neon1300** [reply]
原来是这样啊  (汗一个) 

感谢大家的回复 <(_ _)>
